atlasdb-tests-shared/src/main/java/com/palantir/timelock/paxos/AbstractInMemoryTimelockExtension.java

```diff
@@ -159,6 +167,10 @@ private void createHelperServices(MetricsManager metricsManager) {
         RedirectRetryTargeter redirectRetryTargeter = timeLockAgent.redirectRetryTargeter();
         ConjureTimelockService conjureTimelockService = ConjureTimelockResource.jersey(
                 redirectRetryTargeter, (_namespace, _context) -> delegate.getTimelockService());
+        MultiClientConjureTimelockService multiClientConjureTimelockService = MultiClientConjureTimelockResource.jersey(
+                redirectRetryTargeter, (_namespace, _context) -> delegate.getTimelockService());
+        internalMultiClientConjureTimelockService =
+                new InMemoryInternalMultiClientConjureTimelockService(multiClientConjureTimelockService);
```

Comment:
What is this for? Why can't we just delegate to normal timelock?

Reply:
It doesn't have a multi client thing which is needed for the batchers

This is because we do not have a single client endpoint

Unless I'm missing something